.changes/next-release/enhancment-cloudformation-66951.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "enhancment",
```

Comment:
enhancment -> enhancement. On that note, I'm not sure if enhancement is the right label here. It is more of a breaking change. I'm also fine holding off on the changelog entry for now as we will likely document all of these breaking changes in the Changelog when v2 goes GA.